### DISCUSSION TITLE
Best hotel you ever visited

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#1007] [2023 Attraction Contributions]
Please share your experience
I am planning my next vacation now and would like to have some inspiration

**Comment by MirkosMangle** [comment]
I stayed in many awesome hotels , especially for the last couple of years.
Best experience i had so far with the [red lodge mountain customer service](https://www.pissedconsumer.com/company/red-lodge-mountain/customer-service.html), as they are always so friendly.
I think a decent room/bathroom, great food and scenic view is not the most important in the hospitality.
